openssl.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.3.0"
 
+  spec.add_runtime_dependency 'ipaddr'
```

Comment:
Use double quotes. The style is not consistent throughout the code base sadly, but the file is using double quotes for all other string literals.

Reply:
Fixed. Sorry about that — I should have paid more attention before I cut-and-pasted.